### ISSUE TITLE
Remove the FAQ from doc/core/howto, since it is a duplicate of the FAQ wiki page

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3236]
|<img alt="tlesher's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| tlesher reported|
|-|-|
|Trac ID|trac#3236|
|Type|defect|
|Created|2008-05-14 13:59:48Z|
|Branch|https://github.com/twisted/twisted/tree/drop-faq-3236|

FAQ entry http://twistedmatrix.com/projects/core/documentation/howto/faq.html#auto35 provides a link to "where the Twisted team hangs out", http://twistedmatrix.com/services/online-help.  The website returns a 404 for this page, containing the test "no such resource".

Should probably point to a real page, lest people interpret the URL and the resulting text as subtle commentary.

<details><summary>Searchable metadata</summary>

```
trac-id__3236 3236
type__defect defect
reporter__tlesher tlesher
priority__normal normal
milestone__ 
branch__branches_drop_faq_3236 branches/drop-faq-3236
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1210773588000000 1210773588000000
changetime__1226848125000000 1226848125000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

Someone transcribed the FAQ onto the wiki.  It's probably a very bad idea to maintain these two documents independently.  Loathesome though wikis may be, I suggest dropping `doc/core/howto/faq.xhtml`.  That means we will no longer be distributing the FAQ, which is a regression of sorts, which will need a solution of its own.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

(In [24891]) Branching to 'drop-faq-3236'

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

Removed the faq document in r24892

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Making the ticket summary more accurately describe what is happening.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set owner to @thijstriemstra|
|-|-|

For some reason, this makes the doc builder unhappy.  It seems there are some undefined references now.  It's a bit hard to read the output on buildbot: <http://buildbot.twistedmatrix.com/builders/documentation/builds/844/steps/process-docs/logs/stdio>.  Running the BookBuilder locally might be easier to debug.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_7"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Ok so I need need to figure out how to run the BookBuilder locally.. will go through the docs and if I can't find those instructions I'll report back here.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_8"></a>|@thijstriemstra removed owner<br>@thijstriemstra set status to `new`|
|-|-|

There was still a reference in book.text which I removed in r25355, solving the problem cause the docbuilder now succeeds: http://buildbot.twistedmatrix.com/builders/documentation/builds/103

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_9"></a>|@therve set owner to @thijstriemstra|
|-|-|

Please open a bug to distribute the FAQ again, then merge.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_10"></a>|@thijstriemstra set status to `assigned`|
|-|-|

Replying to [therve](#note_9):
> Please open a bug to distribute the FAQ again, then merge.

Opened [#3539](https://github.com/twisted/twisted/issues/3539)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_11"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [25375]) Merge drop-faq-3236: Remove the FAQ from doc/core/howto, since it is a duplicate of the FAQ wiki page.

Author: thijs
Reviewer: exarkun, therve
Fixes: [#3236](https://github.com/twisted/twisted/issues/3236)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
